src/pykeen/losses.py

```diff
@@ -140,6 +140,8 @@
 from torch.nn import functional
 from torch.nn.modules.loss import _Loss
 
+from pykeen.training.utils import apply_label_smoothing
```

Comment:
will have to move that into the more general `pykeen.utils` to avoid circular importing

Reply:
After the changes, it is only used in `losses.py`.